client/stylesheets/_toCat.scss

```diff
@@ -3605,12 +3608,13 @@ On click the images will be displayed at normal size to complete the effect
 
 .profileUserInfo > ul li {
     display: inline-block;
+    vertical-align: middle !important;
 }
 
 .profileUserInfo ul li a {
     padding: 2px 5px;
-    margin-left: 30px;
     font-size: 1.6rem;
+    vertical-align: middle !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces